### ISSUE TITLE
Multiple selections within one event

### ISSUE THREAD

**Issue by sgeis** [givingforum/givingforum#178]
We have several series of events at IPA that we have had to use a different registration system for (Certain) because of series pricing and registration options. Here is an example of one of our series:

·There are 6 webinars in the series

·Members and non-members should have the option to register for the full webinar series or to pick and choose as many or as few webinars as they would like.

·The cost to register for the full series is $150 for members and $225 for non-members. For some series, there might be a discounted rate for members instead of a fixed price.

·The per webinar cost is $35 for members and $55 for non-members

·We’d like registrants to be able to register for as many or as few webinars as they want from one event page on the website.

·Members should only have to add their selection to the cart once or click the checkout button once.

·Currently, we had to set up 6 separate events in Drupal and link each one to Certain, our other registration system.

We are curious is anyone else is interested in making registration for a series of events easier. We don’t want to spend a lot of money or time of this but would like to explore if there is an easier way to manage registration for a series of events.

**Comment by allenlaurie** [comment]
CCP is interested.

[cid:image005.jpg@01D03ADB.1B3A5180]
Laurie Allen | Communications Manager
Connecticut Council for Philanthropy
860-525-5585 | lallen@CTphilanthropy.org<mailto:lallen@CTphilanthropy.org>
221 Main Street, Hartford, CT  06106

CONNECT WITH US: Website<http://www.ctphilanthropy.org/> | Facebook<http://www.facebook.com/CTphilanthropy> | Twitter<http://www.twitter.com/CTphilanthropy> | News<http://www.ctphilanthropy.org/s_ccp/sec.asp?CID=6336&DID=13845> | Blog<http://ctphilanthropynotes.org/>

[ccp1_294]

**Comment by EmilyBaron** [comment]
GWP is not interested

**Comment by PhilOhio** [comment]
POH is interested. We are in the middle of a program series as well and it was not easy to set up.

**Comment by jonstahl** [comment]
PNW is not interested at this time.

**Comment by DNTownsend** [comment]
We don't really offer event "series," so probably not something we'd be interested in now. However, simplification of the registration system is of interest to us, particularly making the process more nimble for both admin and users.

**Comment by vrozansky** [comment]
Tauno updated this ticket after a VERY long time... It's somewhat encouraging that more work on the registration module is currently underway and we'd stand to benefit from it in the future. It doesn't address any of the immediate needs, of course.

@jonstahl - he's referencing the work he's done for MIE, so perhaps you can shed some light on this.

Here's his response:

"@vrozansky This sounds pretty similar to work we (I) did a long time ago for MIE. I'm not sure if this functionality is still being used by them or if it really fits with what you're looking for. We're also expecting a considerable amount of registration work coming up here in the next 3-6 months. Some of this might help address this use case as well."

**Comment by jonstahl** [comment]
MIE’s registration functionality is non-functional at present. ☹

**Comment by mtrimbach** [comment]
CNJG only has a need for something like this perhaps once a year.  I get around it by having multiple registrations types for each of the different options which is do-able but by no means ideal.  Would possibly have an interest depending on cost.

**Comment by vrozansky** [comment]
We had a discussion about handling series at the user's group in Baltimore. @wcorkill demonstrated how CMF handles this on their site by creating a separate page (example: https://www.michiganfoundations.org/education/cmf-jcp-grantmaking-series). He also demonstrated the registration setup for these series.

I'll leave this ticket open until August 10th so folks can ask any questions or Bill can provide further details. Any instructions will be captured and added to our Wiki.

Also, Tauno mentioned that additional enhancements are in the pipeline in the next 3-6 months, so it might make sense to wait for them too. I'll resolve this ticket after August 10th, but we can re-open it or create a new ticket if there's still interest in customizing a solution for this issue. Thank you.

**Comment by EmilyBaron** [comment]
GWP doesn't have a need for this kind of advanced program registration right now.

**Comment by sgeis** [comment]
Indiana is fine with waiting on the additional enhancements.

**Comment by MainePhilanthropyCenter** [comment]
MPC we would like to wait for the enhancements in the next few months.

**Comment by PhilOhio** [comment]
POH can wait for future enhancements

**Comment by vrozansky** [comment]
Resolving this ticket with a Parking Lot tag so we can revisit at a later time. Feel free to re-open for any new developments or requests. Thank you.
